src/store/modules/chatMessages.js

```diff
@@ -21,13 +21,12 @@ const getters = {
         }
         return 0;
       });
-      return conversation;
     }
-    return null;
+    return cm;
   },
-  getChatMessageObject(state) {
-    // state[0] will change to the selected inbox item
-    return state.chatMessages[0];
+  getChatMessageById: (state, getters) => (id) => {
+    let cm = state.chatMessages.find((chatMessage) => chatMessage.id === id);
+    return cm;
```

Comment:
this function could be changed to:
`getChatMessageById: (state) => (id) => state.chatMessages.find((chatMessage) => chatMessage.id === id) || {}`

This way, we return an empty object in the case of undefined. (its also short making it look like we know what we are doing)

Reply:
I don't want to return an empty object because it is harder to handle then checking for undefined. I did clean up the syntax

Reply:
This is tottally personal preferences but you dont need to have the `return` key word or the `{` `}` brackets after the fat arrow.

Reply:
Personally I find it more readable with brackets and the return but am fine with it either way